src/js/entry.js

```diff
@@ -84,7 +84,7 @@ export function chromeInit(libjwt) {
         identifyApp: (data) => Promise.all([jwtResolver, navResolver]).then(
             () => identifyApp(data, store.getState().chrome.globalNav)
         ),
-        navigation: appNav,
+        navigation: () => console.error('Don\'t use this function, it has been deprecated!'),
```

Comment:
Can you include the name of the function here?

`Don't use insights.chrome.navigation, it has been deprecated`?